src/interactivity/base/InteractivityFactory.cpp

```diff
@@ -29,6 +29,88 @@
 using namespace std;
 using namespace Microsoft::Console::Interactivity;
 
+HRESULT PseudoConsoleWindowAccessibilityProvider::RuntimeClassInitialize(HWND pseudoConsoleHwnd) noexcept
+{
+    RETURN_HR_IF_NULL(E_INVALIDARG, pseudoConsoleHwnd);
+    _pseudoConsoleHwnd = pseudoConsoleHwnd;

```

Comment:
is there any compelling need for the PCWAP to keep a handle to its owning window?

Reply:
Just this: https://github.com/microsoft/terminal/blob/767cabfa5638f43a8aa15cadcbf5209338e6f25c/src/interactivity/base/InteractivityFactory.cpp#L107-L112